fstest/file.go

```diff
@@ -653,6 +654,7 @@ func TestFileSync(tb testing.TB, o FSOptions) {
 	assert.NoError(tb, file.Close())
 }
 
+//nolint:govet
```

Comment:
It's at least awkward in the golangci config, if not impossible - it lets you disable whole linters (i.e. `govet` in this case), but not sub-linters like `fieldalignment`. 

If you want to avoid linting all tests with `govet`, then that's an option, but...